src/main/drivers/serial_uart_stm32f30x.c

```diff
@@ -113,6 +140,51 @@ static void handleUsartTxDma(dmaChannelDescriptor_t* descriptor)
 }
 #endif
 
+static uartDevice_t uartHardware[] = {
+    // USART1
+    // PA10,PA9
+    // PB7,PB6
+    // PE1,PE0
+    // PC5,PC4
+    {
+        .dev = USART1,
+        .rxDMAChannel = DMA1_Channel5,
+        .txDMAChannel = DMA1_Channel4,
+        .pinPair = {
+            { IO_TAG(PA10), IO_TAG(PA9) },
+            { IO_TAG(PB7), IO_TAG(PB6) },
+#if ((TARGET_IO_PORTC & BIT(5)) && (TARGET_IO_PORTC & BIT(4)))
```

Comment:
Got it.
So, I understand all occurrence of IO_TAG should be DEFIO_TAG_E in this context.